CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Change Log
 
+## [0.8.0](https://github.com/redhat-developer/vscode-xml/milestone/9?closed=1) (July 23, 2019)
+
+### Enhancements
+
+ * `Markdown` support for `completion` and `hover` documentation. See [#32](https://github.com/redhat-developer/vscode-xml/issues/32)
+ * Add completion for `comment` and `#region`. See [lsp4xml#54](https://github.com/angelozerr/lsp4xml/issues/54)
+ * Add completion for `CDATA` block. See [#168](https://github.com/redhat-developer/vscode-xml/issues/168)
+ * Find definition for start/end tag element. See [lsp4xml#535](https://github.com/angelozerr/lsp4xml/issues/535)
+ * Show `relevant XML` completion options based on XML Schema. See [lsp4xm#347](https://github.com/angelozerr/lsp4xml/issues/347) 
+ * Improve `XSD source` information for XML completion. See [lsp4xm#529](https://github.com/angelozerr/lsp4xml/issues/529)
+ * Add support for `textDocument/documentHighlight` for XML Schema types. See [lsp4xm#470](https://github.com/angelozerr/lsp4xml/issues/470)
+ * Add support for `textDocument/completion` for xs:element/@name / xs:extension/@base. See [lsp4xm#451](https://github.com/angelozerr/lsp4xml/issues/451)
+ * Add support for selective outline enablement per file. See [lsp4xm#427](https://github.com/angelozerr/lsp4xml/issues/427)
+ * Parse `.ent` and `.mod` files as DTD files. See [lsp4xm#380](https://github.com/angelozerr/lsp4xml/issues/380)
+ * Add support for `textDocument/typeDefinition` from XML to XMLSchema/DTD. See [lsp4xm#371](https://github.com/angelozerr/lsp4xml/issues/371)
+ * Add support for `textDocument/definition` for XML Schema. See [lsp4xm#148](https://github.com/angelozerr/lsp4xml/issues/148)
+ * Add support for `textDocument/references` for XML Schema types. See [lsp4xm#58](https://github.com/angelozerr/lsp4xml/issues/58)
+ * Add support for `textDocument/codelens` for XML Schema types. See [lsp4xm#55](https://github.com/angelozerr/lsp4xml/issues/55)
+ * Add support for clickable`XSD CodeLens`. See [lsp4xm#490](https://github.com/angelozerr/lsp4xml/issues/490)
+ * Active editor cannot provide outline information. See [#166](https://github.com/redhat-developer/vscode-xml/issues/166)
+ * Add XML support for unsaved XML, XSD and XSL files. See [#156](https://github.com/redhat-developer/vscode-xml/issues/156) 
+ * Trigger `textDocument/didSave` if an XSD file was changed externally from a different editor . See [#132](https://github.com/redhat-developer/vscode-xml/issues/132)
```

Comment:
either this one or bug 131 should be removed, as one is a duplicate of the other

Reply:
`editor.` no space between editor and .